### PR TITLE
add /alias

### PR DESCRIPTION
**Pull request by SolarisFox**
Looks for command aliases and aliases in `data.aliases` to return a list of the aliases.

### CODE REVIEW

**Comment by SolarisFox** [comment]
If this should be in `info.js` as opposed to `commands.js` just let me know and I'll move it.

File: commands.js [hunk 1]
```diff
@@ -2201,6 +2201,58 @@ var commands = exports.commands = {
...
+	aliases: 'alias',
+	alias: function (target, room, user) {
+		if (!target) this.parse('/help alias');
+		target = toId(target);
+
+		if (target in commands) {
+			var targetCommand;
+			if (typeof commands[target] === 'string') {
+				targetCommand = commands[target];
+			} else if (Array.isArray(commands[target])) {
```
Comment (kotarou3) [inline commands.js]: Move this check up into the `if (target in commands)` statement?

File: commands.js [hunk 2]
```diff
@@ -2201,6 +2201,58 @@ var commands = exports.commands = {
...
+				if (typeof commands[cmd] !== 'string') continue;
+				if (commands[cmd] === targetCommand) aliases.push('/' + cmd);
+			}
+
+			if (aliases.length > 0) return this.sendReplyBox("Aliases for " + target + ":<br>" + aliases.join(', '));
+		} else {
+			var targetAlias;
+			var id;
+			for (var alias in Tools.data.Aliases) {
+				if (alias === target) {
```
Comment (kotarou3) [inline commands.js]: [Yoda condition](http://en.wikipedia.org/wiki/Yoda_conditions)?

File: commands.js [hunk 3]
```diff
@@ -2201,6 +2201,58 @@ var commands = exports.commands = {
...
+					targetAlias = Tools.data.Aliases[alias];
+					id = toId(Tools.data.Aliases[alias]);
+					break;
+				} else if (target === toId(Tools.data.Aliases[alias])) {
```
Comment (kotarou3) [inline commands.js]: [Yoda condition](http://en.wikipedia.org/wiki/Yoda_conditions)?

File: commands.js (hunk 2 above)
Comment (kotarou3) [inline commands.js]: A quicker way to check for this is just `Tools.data.Aliases[target]`

**Comment by Zarel** [comment]
I don't see a need for this?

In theory, the help entry should give all the names for the aliases we want people to use.

Other aliases we don't want people to use and would like to deprecate and remove if possible.

**Comment by SolarisFox** [comment]
Yeah okay, the help entries having them makes sense to me, I'll convert this pullreq to that.

**Comment by Zarel** [comment]
Um... I'm worried. What do you mean by that?

**Pull request closed without merging**